.github/workflows/ctf.yaml

```diff
@@ -77,6 +84,10 @@ jobs:
           body: |
             Start a new ephemeral environment with changes proposed in this pull request:
 
+            Fedora Environment
             [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#${{ github.server_url }}/${{ github.repository }}/pull/${{ github.event.pull_request.number }})
 
+            Oracle Linux 8 Environment
```

Comment:
Do we want to add SLES or any Debian based distributions?

Reply:
Yes, but then we need to add dockerfiles for them. It should not be difficult to create them but the maintainer of the distro could add them. If I find some time I can try.

Reply:
Sounds good, I just wanted to make sure we had an answer to that question. We merged this PR without those.